.github/workflows/python.yml

```diff
@@ -312,6 +312,14 @@ jobs:
           destination: "rerun-example-rrd/version/${{github.ref_name}}"
           parent: false
 
+      - name: "Upload web-viewer (latest release)"
+        if: github.ref == 'latest'
+        uses: google-github-actions/upload-cloud-storage@v1
+        with:
+          path: "rrd"
+          destination: "rerun-example-rrd/web_viewer"
```

Comment:
Maybe name this “example-rrd/latest”?

Reply:
There was already a folder in the bucket named `web_viewer` which is what I assumed `https://app.rerun.io/` was pointing to, but now I don't know where `https://app.rerun.io/` is pointing